integration/volume/volume_test.go

```diff
@@ -104,6 +105,19 @@ func TestVolumesInspect(t *testing.T) {
 	createdAt, err := time.Parse(time.RFC3339, strings.TrimSpace(inspected.CreatedAt))
 	assert.NilError(t, err)
 	assert.Check(t, createdAt.Unix()-now.Unix() < 60, "CreatedAt (%s) exceeds creation time (%s) 60s", createdAt, now)
+
+	// update atime and mtime for the "_data" directory (which would happen during volume initialization)
+	modifiedAt := time.Now().Local().Add(5 * time.Hour)
+	err = os.Chtimes(inspected.Mountpoint, modifiedAt, modifiedAt)
+	assert.NilError(t, err)
+
+	inspected, err = client.VolumeInspect(ctx, vol.Name)
+	assert.NilError(t, err)
+
+	createdAt2, err := time.Parse(time.RFC3339, strings.TrimSpace(inspected.CreatedAt))
+	assert.NilError(t, err)
+
+	assert.Equal(t, createdAt, createdAt2)
```

Comment:
I'd like a comment here describing what we're checking for in a little more detail (that changing the `_data` mtime does not affect `CreatedAt`).

Reply:
Added a small comment